.grit/patterns/python/json-response-vs-json.md

```diff
@@ -0,0 +1,57 @@
+---
+title: use `JsonResponse` over `json` and `HttpResponse`
+tags: [fix, best-practice, django]
+---
+
+Use `JsonResponse` over `json` and `HttpResponse`
```

Comment:
Please add some more explanation. Why is this preferred / valued?

Reply:
Hi @morgante, updated it now